pycds/orm/tables.py

```diff
@@ -297,6 +298,9 @@ class Variable(Base):
         UniqueConstraint(
             "network_id", "net_var_name", name="network_variable_name_unique"
         ),
+        CheckConstraint(
+            "net_var_name !~ '\W'", name="ck_net_var_name_valid_identifier"
```

Comment:
Can this be simplified by removing the implicit double negative? `net_var_name ~ '\w'` (lowercase `w`)